src/frontend/src/btc/components/send/BtcReviewNetwork.svelte

```diff
@@ -0,0 +1,17 @@
+<script lang="ts">
+	import { BTC_REGTEST_NETWORK, BTC_TESTNET_NETWORK, BTC_MAINNET_NETWORK } from '$env/networks.env';
+	import ReviewNetwork from '$lib/components/send/ReviewNetwork.svelte';
+	import type { NetworkId } from '$lib/types/network';
+	import { isNetworkIdBTCTestnet, isNetworkIdBTCRegtest } from '$lib/utils/network.utils';
+
+	export let networkId: NetworkId | undefined = undefined;
```

Comment:
Shouldn't we call it `destinationNetworkId` or the consumer is aware that `networkId` is the destination?

Reply:
Since underlying `ReviewNetwork` is the first component in the tree who works with both source and destination networks, I'm passing `networkId` as `destinationNetworkId` to avoid any confusions.

Reply:
Thanks for the explanation